webpack/index.js

```diff
@@ -0,0 +1,16 @@
+import ReactDOM from 'react-dom';
+import Navbar from './scenes/Nav';
+
+let attempts = 0;
+
+function mountReact() {
```

Comment:
This seems quite complicated and unreliable. Can't we just
```js
document.addEventListener("DOMContentLoaded", function(){ });
```
?

Reply:
can you please just use the react mounter that exists in core instead ? :)

Reply:
@ohadlevy @tstrachota yes and yes. This code needs to be refactored. I'll look at the mounting service from foreman. We do have the timing issue so we may need to have both the `DOMContentLoaded` event followed by the core mounter unless the core mounter handles waiting for the DOM node to be available.

Reply:
@danseethaler it does -> see deffer at https://github.com/theforeman/foreman/blob/develop/app/helpers/reactjs_helper.rb#L3

Reply:
Awesome! Thanks @ohadlevy! I'll make it happen.

Reply:
@ohadlevy can you tell me how to use the defer option you mentioned in that link? I don't see any way to invoke that with the [mounting service](https://github.com/theforeman/foreman/blob/51919636bad9ba57586c1c983f339747bb67f0ee/webpack/assets/javascripts/react_app/common/MountingService.js).

I'm running into the same issue where the DOM node I want to mount to isn't available when my js runs.

Reply:
Defer is an attribute of the <script> tag, placing it here https://github.com/Katello/katello/pull/6992/files#r145622363 should do the magic.